module/os/freebsd/spl/spl_uio.c

```diff
@@ -45,6 +45,15 @@
 #include <sys/vnode.h>
 #include <sys/zfs_znode.h>
 
+/* XXX: should be an __FreeBSD_version check once API is upstream. */
```

Comment:
The freeuio() interface is upstream already, and this bug is fixed in FreeBSD's copy of ZFS, but not in OpenZFS. I'll get that fixed (using a __FreeBSD_version check).

Reply:
Oops, I misread, this is not fixed in FreeBSD (but that's harmles for now).